tests/transfers/test_upload.py

```diff
@@ -1,42 +1,33 @@
-import sqlite3
 import pytest
-import json
+from unittest.mock import MagicMock
 import boto3
 from moto import mock_s3
 import os
 import slapp.transfers.upload as up
 
 
-@pytest.fixture(scope='module')
-def db_file(tmpdir_factory):
-    db = tmpdir_factory.mktemp("upload").join("mydatabase.db")
-    conn = sqlite3.connect(db)
-    conn.execute("CREATE TABLE manifest_table (manifest text)")
-    conn.close
-    yield db
-
+@pytest.fixture
+def mock_db_conn_fixture(request, tmpdir_factory):
```

Comment:
No need for `request` if you are not indirectly parametrizing this `mock_db_conn_fixture`.

Reply:
removed `request`